owncloudApp/src/main/java/com/owncloud/android/presentation/sharing/shares/PublicShareDialogFragment.kt

```diff
@@ -789,6 +925,7 @@ class PublicShareDialogFragment : DialogFragment() {
         private const val ARG_ACCOUNT = "ACCOUNT"
         private const val ARG_DEFAULT_LINK_NAME = "DEFAULT_LINK_NAME"
         private const val KEY_EXPIRATION_DATE = "EXPIRATION_DATE"
+        private const val SPECIALS_CHARACTERS = "\"!\"#$%&'()*+,-./:;<=>?@[\\]^_`{|}~\""
```

Comment:
No, not done. I told you there is twice the character `"` and I keep seeing it twice escaped. So?

Reply:
Done